### ISSUE TITLE
Moonlight stream is crashing when i switch the game to full screen mode

### ISSUE THREAD

**Issue by astar10239** [LizardByte/Sunshine#290]
### Describe the Bug

Hi,

I am using AWS EC2 with Windows 10 Server 2019 and AMD V520.

I am successfully streaming sunshine fine, but i have to run the game in window mode.

When i try to switch the game to full screen, Moonlight stream crashes and in the sunshine server i get this error

`failed to create input shader [0x80070057]`

A detailed log

```txt
--end relative mouse move packet--
[2022:07:31:11:17:57]: Debug: --begin keyboard packet--
keyAction [03]
keyCode [8020]
modifiers [00]
--end keyboard packet--
[2022:07:31:11:17:57]: Debug: --begin keyboard packet--
keyAction [04]
keyCode [8020]
modifiers [00]
--end keyboard packet--
[2022:07:31:11:17:57]: Debug: Detecting monitors...
[2022:07:31:11:17:57]: Debug:
====== ADAPTER =====
Device Name      : AMD Radeon Pro V520 MxGPU
Device Vendor ID : 0x00001002
Device Device ID : 0x00007362
Device Video Mem : 8011 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 4095 MiB

    ====== OUTPUT ======
[2022:07:31:11:17:57]: Debug:     Output Name       : \\.\DISPLAY2
    AttachedToDesktop : yes
    Resolution        : 1920x1080

[2022:07:31:11:17:57]: Debug:
====== ADAPTER =====
Device Name      : Microsoft Basic Render Driver
Device Vendor ID : 0x00001414
Device Device ID : 0x0000008C
Device Video Mem : 0 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 4095 MiB

    ====== OUTPUT ======
[2022:07:31:11:17:57]: Debug:
====== ADAPTER =====
Device Name      : Microsoft Basic Render Driver
Device Vendor ID : 0x00001414
Device Device ID : 0x0000008C
Device Video Mem : 0 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 4095 MiB

    ====== OUTPUT ======
[2022:07:31:11:17:57]: Info:
Device Description : AMD Radeon Pro V520 MxGPU
Device Vendor ID   : 0x00001002
Device Device ID   : 0x00007362
Device Video Mem   : 16011 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 4095 MiB
Feature Level      : 0x0000B100
Capture size       : 1920x1080
Offset             : 0x0
Virtual Desktop    : 1920x1080
[2022:07:31:11:17:57]: Debug: Source format [DXGI_FORMAT_R8G8B8A8_UNORM]
[2022:07:31:11:17:57]: Error: Failed to create input shader resource view [0x80070057]
[2022:07:31:11:17:57]: Debug: --begin relative mouse move packet--
deltaX [0]
deltaY [1]
--end relative mouse move packet--
[2022:07:31:11:17:57]: Debug: --begin relative mouse move packet--
deltaX [1]
deltaY [1]
--end relative mouse move packet--
[2022:07:31:11:17:57]: Debug: Waiting for video to end...
[2022:07:31:11:17:57]: Debug: Waiting for audio to end...
[2022:07:31:11:17:57]: Debug: Waiting for control to end...
[2022:07:31:11:17:57]: Debug: Resetting Input...
[2022:07:31:11:17:57]: Debug: Removing references to any connections...
[2022:07:31:11:17:57]: Debug: Session ended
[2022:07:31:11:17:57]: Debug: Waiting for main listening thread to end...
[2022:07:31:11:17:57]: Debug: Waiting for main video thread to end...
[2022:07:31:11:17:57]: Debug: Waiting for main audio thread to end...
[2022:07:31:11:17:57]: Debug: Waiting for main control thread to end...
[2022:07:31:11:17:57]: Debug: All broadcasting threads ended
[2022:07:31:11:17:57]: Debug: TUNNEL :: NONE
[2022:07:31:11:17:57]: Debug: METHOD :: GET
[2022:07:31:11:17:57]: Debug: DESTINATION :: /serverinfo
[2022:07:31:11:17:57]: Debug: User-Agent -- Mozilla/5.0
[2022:07:31:11:17:57]: Debug: Accept-Language -- en-IN,*
[2022:07:31:11:17:57]: Debug: Accept-Encoding -- gzip, deflate
[2022:07:31:11:17:57]: Debug: Connection -- Keep-Alive
[2022:07:31:11:17:57]: Debug: Host -- ip:47989
[2022:07:31:11:17:57]: Debug:  [--]
[2022:07:31:11:17:57]: Debug: uuid -- 908e2c5d46454c82a6f9ae9e3b838d85
[2022:07:31:11:17:57]: Debug: uniqueid -- 0123456789ABCDEF
[2022:07:31:11:17:57]: Debug:  [--]
[2022:07:31:11:17:57]: Info: /CN=NVIDIA GameStream Client -- verfied
[2022:07:31:11:17:57]: Debug: TUNNEL :: HTTPS
[2022:07:31:11:17:57]: Debug: METHOD :: GET
[2022:07:31:11:17:57]: Debug: DESTINATION :: /serverinfo
[2022:07:31:11:17:57]: Debug: User-Agent -- Mozilla/5.0
[2022:07:31:11:17:57]: Debug: Accept-Language -- en-IN,*
[2022:07:31:11:17:57]: Debug: Accept-Encoding -- gzip, deflate
[2022:07:31:11:17:57]: Debug: Connection -- Keep-Alive
[2022:07:31:11:17:57]: Debug: Host -- ip:47984
 
```

### Sunshine Host Operating System and Version

Windows 10

### Architecture

Windows 2019 Server

### Sunshine Version

0.14.0

### GPU Type

AMD

### GPU Model

AMD V520

### GPU Driver/Mesa Version

Latest

**Comment by astar10239** [comment]
Seems like its happening on games, when i try to make Destiny 2 full screen or NFS heat-wire it breaks the stream

**Comment by astar10239** [comment]
Update - Happening on 3060 on Destiny 2 on full screen now seems like

**Comment by nlemberger** [comment]
I have what appears to be the same problem on Windows 11, with an nVidia RTX2060.  The error occurs when attempting to launch Mass Effect: Andromeda in full-screen.  It does play fine windowed.

```
[2022:09:30:23:35:23]: Info: CLIENT CONNECTED
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Debug: type [IDX_REQUEST_IDR_FRAME]
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Debug: type [IDX_START_B]
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Verbose: type [IDX_START_A]
[2022:09:30:23:35:23]: Verbose: Recv: 172.29.255.167:36745 :: VIDEO
[2022:09:30:23:35:23]: Debug: RAISE: 172.29.255.167:36745 :: VIDEO
[2022:09:30:23:35:23]: Debug: Received ping from 172.29.255.167:36745 [474E4950]
[2022:09:30:23:35:23]: Debug: Start capturing Video
[2022:09:30:23:35:23]: Debug: Detecting monitors...
[2022:09:30:23:35:23]: Debug:
====== ADAPTER =====
Device Name      : NVIDIA GeForce RTX 2060
Device Vendor ID : 0x000010DE
Device Device ID : 0x00001F15
Device Video Mem : 5980 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 7870 MiB

    ====== OUTPUT ======
[2022:09:30:23:35:23]: Debug:     Output Name       : \\.\DISPLAY1
    AttachedToDesktop : yes
    Resolution        : 1920x1080

[2022:09:30:23:35:23]: Debug:
====== ADAPTER =====
Device Name      : Microsoft Basic Render Driver
Device Vendor ID : 0x00001414
Device Device ID : 0x0000008C
Device Video Mem : 0 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 7870 MiB

    ====== OUTPUT ======
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Debug: --begin relative mouse move packet--
deltaX [1]
deltaY [1]
--end relative mouse move packet--
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Debug: --begin relative mouse move packet--
deltaX [-1]
deltaY [-1]
--end relative mouse move packet--
[2022:09:30:23:35:23]: Info:
Device Description : NVIDIA GeForce RTX 2060
Device Vendor ID   : 0x000010DE
Device Device ID   : 0x00001F15
Device Video Mem   : 5980 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 7870 MiB
Feature Level      : 0x0000B100
Capture size       : 1920x1080
Offset             : 0x0
Virtual Desktop    : 1920x1080
[2022:09:30:23:35:23]: Debug: Source format [DXGI_FORMAT_R10G10B10A2_UNORM]
[2022:09:30:23:35:23]: Error: Failed to create input shader resource view [0x80070057]
[2022:09:30:23:35:23]: Verbose: type [IDX_ENCRYPTED]
[2022:09:30:23:35:23]: Verbose: type [IDX_START_A]
[2022:09:30:23:35:23]: Verbose: Recv: 172.29.255.167:36111 :: AUDIO
[2022:09:30:23:35:23]: Debug: RAISE: 172.29.255.167:36111 :: AUDIO
[2022:09:30:23:35:23]: Debug: Received ping from 172.29.255.167:36111 [474E4950]
[2022:09:30:23:35:23]: Debug: Start capturing Audio
[2022:09:30:23:35:23]: Debug: Couldn't initialize audio client for [Mono]: [0x88890008]
[2022:09:30:23:35:23]: Debug: Mono: unsupported
[2022:09:30:23:35:23]: Debug: Stereo: supported
[2022:09:30:23:35:23]: Verbose: ===== Device =====
Device ID          : {0.0.0.00000000}.{13afd829-d6ed-4f71-8516-665b8646978f}
Device name        : Speakers (Steam Streaming Speakers)
Adapter name       : Steam Streaming Speakers
Device description : Speakers

[2022:09:30:23:35:23]: Debug: Couldn't initialize audio client for [Mono]: [0x88890008]
[2022:09:30:23:35:23]: Debug: Mono: unsupported
[2022:09:30:23:35:23]: Debug: Waiting for video to end...
[2022:09:30:23:35:23]: Debug: Waiting for audio to end...
[2022:09:30:23:35:23]: Debug: Stereo: supported
[2022:09:30:23:35:23]: Verbose: ===== Device =====
Device ID          : {0.0.0.00000000}.{15c25665-624b-4a75-8514-406d5012ed7d}
Device name        : Speakers (Steam Streaming Microphone)
Adapter name       : Steam Streaming Microphone
Device description : Speakers

[2022:09:30:23:35:24]: Debug: Couldn't initialize audio client for [Mono]: [0x88890008]
[2022:09:30:23:35:24]: Debug: Mono: unsupported
[2022:09:30:23:35:24]: Debug: Stereo: supported
[2022:09:30:23:35:24]: Verbose: ===== Device =====
Device ID          : {0.0.0.00000000}.{51b3f395-6572-49df-945f-531b38658b08}
Device name        : Speakers (Realtek(R) Audio)
Adapter name       : Realtek(R) Audio
Device description : Speakers

[2022:09:30:23:35:24]: Debug: Trying audio format [Mono]
[2022:09:30:23:35:24]: Debug: Couldn't initialize audio client for [Mono]: [0x88890008]
[2022:09:30:23:35:24]: Debug: Trying audio format [Stereo]
[2022:09:30:23:35:24]: Debug: Found audio format [Stereo]
[2022:09:30:23:35:24]: Debug: Waiting for control to end...
[2022:09:30:23:35:24]: Debug: Resetting Input...
[2022:09:30:23:35:24]: Debug: Removing references to any connections...
[2022:09:30:23:35:24]: Debug: Session ended
[2022:09:30:23:35:24]: Debug: Waiting for main listening thread to end...
[2022:09:30:23:35:24]: Debug: Waiting for main video thread to end...
[2022:09:30:23:35:24]: Debug: Waiting for main audio thread to end...
[2022:09:30:23:35:24]: Debug: Waiting for main control thread to end...
[2022:09:30:23:35:24]: Debug: All broadcasting threads ended
```

Windows 11 Pro - 22H2
Nvidia Gforce RTX 2060, Driver v517.48
Sunshine 0.14.1

**Comment by cgutman** [comment]
This was fixed by #609
